app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,47 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+section {
+  overflow: auto;
+}
+.center {
+  text-align: center;
+  vertical-align: center;
+}
+.center h1 {
+	margin-top: 180px;
+  margin-bottom: 30px;
+  color: #000000;
+}
+.jumbotron{
+	background: url(eventhero.jpg);
+	background-size: 100%;
+	height: 100%;
+	margin-bottom: 0px;
```

Comment:
Line should be indented with spaces, not tabs<br>`0px` should be written without units as `0`